Makefile

```diff
@@ -75,9 +76,18 @@ generate: controller-gen mockgen
 docker-build: test
 	docker build . -t ${IMG}
 
-# Push the docker image
-docker-push:
-	docker push ${IMG}
+# Build the image with podman
+podman-build: test
+	podman build . -t ${IMG}
+
+# Push the image with podman
+podman-push:
+	podman push ${IMG} --tls-verify=false
```

Comment:
maybe instead make the binary (podman / docker) be an env variable and should be set when you run the command
binary can be called $CONTAINER_ENGINE

Reply:
nvm, podman is better in some ways and I will need to create docker variant for it anyways